src/appMain/CMakeLists.txt

```diff
@@ -217,7 +217,8 @@ endif ()
 # Install rules
 install(TARGETS ${PROJECT} DESTINATION bin)
 install(
-  FILES build_config.txt log4cxx.properties audio.8bit.wav test.txt smartDeviceLink.ini
+  FILES build_config.txt log4cxx.properties audio.8bit.wav test.txt
+    ${CMAKE_CURRENT_BINARY_DIR}/smartDeviceLink.ini
```

Comment:
@SNiukalov Are you sure that SDL version will be written to all places? 
After SDL build you will see the following structure: 
```
├── bin
│   ├── audio.8bit.wav
│   ├── build_config.txt
│   ├── hmi_capabilities.json
│   ├── libappenders.so
│   ├── libapp_service_rpc_plugin.so
│   ├── libPolicy.so
│   ├── librc_rpc_plugin.so
│   ├── libsdl_rpc_plugin.so
│   ├── libUtils.a
│   ├── libvehicle_info_plugin.so
│   ├── log4cxx.properties
│   ├── mycert.pem
│   ├── mykey.pem
│   ├── sample_policy_manager.py
│   ├── sdl_preloaded_pt.json
│   ├── smartDeviceLinkCore
│   ├── smartDeviceLink.ini
│   ├── start.sh
│   └── test.txt
```

as well as the following: 
```
src
│   ├── 3rd_party
│   ├── 3rd_party-static
│   ├── appMain
		   ├── audio.8bit.wav
		   ├── build_config.txt
		   ├── CMakeFiles
		   ├── cmake_install.cmake
		   ├── CTestTestfile.cmake
		   ├── hmi_capabilities.json
		   ├── libPolicy.so
		   ├── log4cxx.properties
		   ├── Makefile
		   ├── sample_policy_manager.py
		   ├── sdl_preloaded_pt.json
		   ├── smartDeviceLinkCore
		   ├── smartDeviceLink.ini
		   ├── start_external_proprietary.sh
		   ├── start.sh
		   ├── test
		   └── test.txt

│   ├── components
│   └── plugins
```
So `smartDeviceLink.ini` is present in 2 places. Check & fix if required please.

Reply:
@AByzhynar 
Yes, CMAKE_CURRENT_BINARY_DIR - path to src->appMain where we write sdl version:https://github.com/smartdevicelink/sdl_core/blob/a9951dcb58c759686c108799b5c530b9470c4a93/src/appMain/CMakeLists.txt#L186-L193

After that we copy this file in the bin  folder here: https://github.com/smartdevicelink/sdl_core/blob/a9951dcb58c759686c108799b5c530b9470c4a93/src/appMain/CMakeLists.txt#L218-L225

Reply:
@SNiukalov :+1: